proofs/interface.go

```diff
@@ -46,3 +48,11 @@ const (
 	// Test configures the SectorBuilder to be used with large sectors, in tests.
 	Test
 )
+
+func GetSectorStoreType() SectorStoreType {
```

Comment:
BLOCKING: I don't think this function belongs here. I think that it should be part of the `gengen` package and should not be exported. I think folks need to be creating a query message to get this value from the `StorageMarketActor`, via a query message, if they need it at runtime.